src/IonDecimal.ts

```diff
@@ -129,28 +128,39 @@ export class Decimal {
     }
 
     /**
-     * Constructor helper shared by the public constructor and _fromJsbiCoefficient.
-     * @hidden
+     * Given a string containing the Ion text representation of a decimal value,
+     * returns a new Decimal object corresponding to the value.  If a string such as
+     * '5' is provided, a 'd0' suffix is assumed.
      */
-    private _initialize(isNegative: boolean, coefficient: JSBI, exponent: number) {
-        this._isNegative = isNegative;
-        this._coefficient = coefficient;
-        // If the exponent is -0, set it to 0, which is equivalent.
-        // See: https://github.com/amzn/ion-js/issues/474
-        if (Object.is(-0, exponent)) {
-            exponent = 0;
+    static parse(str: string): Decimal {
```

Comment:
whats going on here? the diff looks off...

Reply:
the formatter moved the `_initialize` method below (to where your second comment is), and moved the `parse` method up to this location.  i suspect it's a rule about moving static methods toward the top of a class